components/scripts/lib/cli-parsers/common.m4

```diff
@@ -3,7 +3,7 @@
 # ARG_OPTIONAL_BOOLEAN([enable-gradle-enterprise],[e],[])
 # ARG_OPTIONAL_SINGLE([git-branch],[b],[])
 # ARG_OPTIONAL_SINGLE([git-commit-id],[c],[])
-# ARG_OPTIONAL_SINGLE([git-options],[o],[])
+# ARG_OPTIONAL_SINGLE([git-options],[o],[],[--depth=1])
```

Comment:
@erichaagdev @jthurne just curious: what impact does the declaration of `depth` on this line have?

Reply:
That sets the default value for git-options (when the scripts are used non-interactively).